utils/log.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import logging.config
+import sys
+import os
+
+def logPath() -> str:
```

Comment:
we should avoid redeclaration.

would it be better to move from Data related functions to Variables in a separate file like you did for LOG_CONFIG?

If so, I suggest creating a config.py file instead of log.py to hold those values. 

Initially, I wanted to avoid global variables, so I used functions to return paths.

Reply:
but log.py also has the StreamLogger class.

So i move everything config related to the config.py and everything else stays the same